Sepia floor tiles fly slowly, yet further

81Denton opened the pull request:
:cl: Denton
tweak: Sepia floor tiles fly really slowly, yet far when thrown. Don't hit yourself!
/:cl:

"Time seems to flow very slowly around these tiles."

Sepia floor tiles now fly so slowly that you can easily run past one after throwing it and even hit yourself. They should also fly four times as far.

ExcessiveUseOfCobblestone commented:
do they make you slow when you walk on them :o?

81Denton commented:
Yeah, the lavaland sloth ruin uses these as floor tiles.

PKPenguin321 commented:
lmao you could throw actually 10 of these on the same tile down a really long hall and just have a slow moving instant death ball

I love it

GunHog commented:
"No one can defeat the Sepia tile attack!" "Jumping...is useless!"

Spaceinaba commented:
little shit like this is my favorite kinda shit ever I love you

MrDoomBringer commented:
![image](https://user-images.githubusercontent.com/29008542/37310228-a50ccd2a-2619-11e8-8aa9-b9cbe7e38b39.png)

GunHog commented:
I..TOO MUCH LAUGHING...I NEED A DOCT-----

81Denton committed:
The flow of time itself is convoluted, with floor tiles centuries old phasing in and out.
HHHH
on second thonk
thanks frozenguy5
and this is... to go even further beyond!
testing tile range

81Denton commented:
@Spaceinaba Thank you! Things like this are what brought me to SS13 in the first place.

MrsTonedOne reviewed (approved):
what could go wrong.
